temporal_sqlalchemy/bases.py

```diff
@@ -194,22 +240,13 @@ def date_modified(self):
 
     @contextlib.contextmanager
     def clock_tick(self, activity: TemporalActivityMixin = None):
-        warnings.warn("clock_tick is going away in 0.5.0",
-                      PendingDeprecationWarning)
-        """Increments vclock by 1 with changes scoped to the session"""
-        if self.temporal_options.activity_cls is not None and activity is None:
-            raise ValueError("activity is missing on edit") from None
-
-        session = orm.object_session(self)
-        with session.no_autoflush:
-            yield self
-
-        if session.is_modified(self):
-            self.vclock += 1
+        warnings.warn("clock_tick is deprecated, assign an activity directly",
+                      DeprecationWarning)
+        if self.temporal_options.activity_cls:
+            if not activity:
+                raise ValueError
+            self.activity = activity
 
-            new_clock_tick = self.temporal_options.clock_model(
-                entity=self, tick=self.vclock)
-            if activity is not None:
-                new_clock_tick.activity = activity
+        yield self
 
-            session.add(new_clock_tick)
+        return
```

Comment:
`return` isn't necessary at the end of a function.

Reply:
I left it in so it looks less confusing to me, but I'm strange that way 😆